hadoop-ozone/interface-storage/src/main/java/org/apache/hadoop/ozone/om/OMMetadataManager.java

```diff
@@ -350,6 +351,10 @@ String getMultipartKey(String volume, String bucket, String key, String
 
   Table<String, String> getTenantUserTable();
 
+  Table<String, OmDBAccessIdInfo> getTenantAccessIdTable();
+
+  Table<String, OmDBAccessIdInfo> getPrincipalToAccessIdsTable();
```

Comment:
Should it be OmDBKerberosPrincipalInfo as the value in the table return type?

Reply:
yup. thanks for pointing that out!